Rakefile

```diff
@@ -217,6 +217,33 @@ namespace :release do
        "Groonga #{version}!!!")
     sh("git", "push", "origin", "v#{version}")
   end
+
+  desc "Post release announces"
+  task :announce do
+    latest_news = Dir.glob("doc/source/news/*.*").max do |a, b|
+      File.basename(a).to_f - File.basename(b).to_f
+    end
+    latest_release_note = File.read(latest_news).split(/^## /)[1]
+    latest_release_note_version = latest_release_note.lines.first[/[\d.]+/]
+    if latest_release_note_version != version
+      raise "release note isn't written"
+    end
+    latest_release_summary =
+      latest_release_note[/^### Summary\s*(.+?)(?=^###|\z)/m, 1]
+    unless latest_release_summary
+      raise "release summary isn't written"
+    end
+    latest_release_date = latest_release_note.lines.first[/\d{4}-\d{2}-\d{2}/]
+    latest_release_major = version.split(".").first
+    latest_release_anchor = "#release-#{version.gsub(".", "-")}"
+    latest_release_url =
+      "https://groonga.org/docs/news/#{latest_release_major}.html"
```

Comment:
Can we use `File.basename(latest_news, ".*")` instead of `latest_release_major`?

Reply:
fix: [296fb5e](https://github.com/groonga/groonga/pull/2291/commits/296fb5ecfb187256902cbefaf5781f5cdd8f54b6)